sabnzbd/deobfuscate_filenames.py

```diff
@@ -97,38 +97,49 @@ def is_probably_obfuscated(myinputfilename):
         return True
 
     # these are signals for the obfuscation versus non-obfuscation
-    capitals = sum(1 for c in filebasename if c.isupper())
-    smallletters = sum(1 for c in filebasename if c.lower())
-    spacesdots = sum(1 for c in filebasename if c == " " or c == ".")
     decimals = sum(1 for c in filebasename if c.isnumeric())
+    upperchars = sum(1 for c in filebasename if c.isupper())
+    lowerchars = sum(1 for c in filebasename if c.islower())
+    spacesdots = sum(1 for c in filebasename if c == " " or c == ".")
 
-    if capitals >= 2 and smallletters >= 2 and spacesdots >= 1:
-        logging.debug("Not obfuscated: capitals >= 2 and smallletters >= 2  and spacesdots >= 1")
+    # Example: "Great Distro"
+    if upperchars >= 2 and lowerchars >= 2 and spacesdots >= 1:
+        logging.debug("Not obfuscated: upperchars >= 2 and lowerchars >= 2  and spacesdots >= 1")
         # useful signs in filebasename, so not obfuscated
         return False
 
-    if spacesdots > 3:
+    # Example: "this is a download"
+    if spacesdots >= 3:
         # useful signs in filebasename, so not obfuscated
-        logging.debug("Not obfuscated: spacesdots > 3")
+        logging.debug("Not obfuscated: spacesdots >= 3")
         return False
 
-    if decimals > 3 and spacesdots > 1:
+    # Example: "Beast 2020"
+    if (upperchars + lowerchars >= 4) and decimals > 3 and spacesdots > 1:
         # useful signs in filebasename, so not obfuscated
-        logging.debug("Not obfuscated: decimals > 3 and spacesdots > 1")
+        logging.debug("Not obfuscated: (upperchars + lowerchars >= 4) and decimals > 3 and spacesdots > 1")
         return False
 
+    # Example: "Catullus", starts with a capital, and most letters are lower case
+    if filebasename[0].isupper() and lowerchars > 2 and upperchars / lowerchars < 0.3:
+        return False
+
+    # Almost last resort
+    """
     # little entropy in the filebasename is a sign of useless names
     if entropy(filebasename) < 3.5:
         logging.debug("Obfuscated: entropy < 3.5")
         return True
+
     # high entropy in the filebasename is a sign of useful name, so not obfuscated
     if entropy(filebasename) > 4.0:
         logging.debug("Not obfuscated: entropy > 4.0")
         return False
+    """
```

Comment:
If this code is not needed, please remove. For history we have Git to always recover code if we need.

Reply:
I'll do!